app/assets/javascripts/channels/comments.js

```diff
@@ -1,34 +1,26 @@
-if ($("meta[name='push_notifications']").length >0) {
+var subscribeToComments = function(){
 
-  $(document).on('click', '.thread-link', function(){
-    if(App.comments){
-      App.comments.unsubscribe();
-    }
-    App.comments = App.cable.subscriptions.create({
-      channel: "CommentsChannel",
-      notification: $(this).attr('href').split('/').pop()},{
```

Comment:
There might be a race condition here, changing `$(this).attr('href').split('/').pop()` to `$(location).attr('href').split('/').pop()` requires that the code is called after the url has changed from `/` to `/notifications/123`, I'm not sure how the order of on click callbacks is defined in jquery, how do we make sure this on click happens after the turbolinks happens after https://github.com/octobox/octobox/blob/master/app/assets/javascripts/application.js#L69